### PR TITLE
[Demo] Open ViViVo links outside iframe

### PR DESCRIPTION
**Pull request by antonymilne**
## Description

@Joseph-Perkins noticed that if you open up [ViViVo on HF in an iframe](https://huggingface.co/spaces/vizro/demo-visual-vocabulary) then the links to github don't work. The reason for this is
https://stackoverflow.com/questions/7649361/github-gist-page-wont-load-in-iframe. The solution is to open these links with ~`target="_top"` to escape the iframe~ `target="_blank"`.

## Notice

- [x] I acknowledge and agree that, by checking this box and clicking "Submit Pull Request":

  - I submit this contribution under the [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0.txt) and represent that I am entitled to do so on behalf of myself, my employer, or relevant third parties, as applicable.
  - I certify that (a) this contribution is my original creation and / or (b) to the extent it is not my original creation, I am authorized to submit this contribution on behalf of the original creator(s) or their licensees.
  - I certify that the use of this contribution as authorized by the Apache 2.0 license does not violate the intellectual property rights of anyone else.
  - I have not referenced individuals, products or companies in any commits, directly or indirectly.
  - I have not added data or restricted code in any commits, directly or indirectly.

### CODE REVIEW

**Review by petar-qb — APPROVED** [review]

**Comment by antonymilne** [comment]
Surprisingly this fix doesn't actually work. So it's blocked pending https://discuss.huggingface.co/t/allow-navigation-outside-iframe/114755/2.

**Review by huong-li-nguyen — APPROVED** [review]

File: vizro-core/examples/visual-vocabulary/custom_components.py [hunk 1]
```diff
@@ -50,7 +50,9 @@ class Markdown(vm.VizroBaseModel):
 
     def build(self):
         """Returns a markdown component with an optional classname."""
-        return dcc.Markdown(id=self.id, children=self.text, dangerously_allow_html=False, className=self.classname)
+        return dcc.Markdown(
+            id=self.id, children=self.text, dangerously_allow_html=False, className=self.classname, link_target="_top"
+        )
```
Comment (antonymilne) [inline vizro-core/examples/visual-vocabulary/custom_components.py:55]:
```suggestion
        return dcc.Markdown(
            id=self.id, children=self.text, dangerously_allow_html=False, className=self.classname, link_target="_blank"
        )
```

**Comment by antonymilne** [comment]
I've raised the issue on the HF discord also but realistically don't expect them to change anything here, so until we hear otherwise the solution is just to open all these links in a new tab with `target="_blank"`. It's not great for usability but unfortunately it's the only way to get the links to open from inside the iframe.

FYI @nadijagraca @huong-li-nguyen this is what I mentioned yesterday at standup.

**Comment by antonymilne** [comment]
HF have said they won't change the settings here so we must continue to open them in a new tab.

**Pull request merged**